packages/core/package.json

```diff
@@ -1245,13 +1246,13 @@
                     "group": "3_lambda@3"
                 },
                 {
-                    "submenu": "aws.submenu.feedback",
-                    "when": "view =~ /^aws\\./",
+                    "submenu": "aws.toolkit.submenu.feedback",
+                    "when": "view =~ /^aws\\./ && view != aws.AmazonQChatView && view != aws.amazonq.AmazonCommonAuth",
```

Comment:
Is it necessary to add `view != aws.AmazonQChatView && view != aws.amazonq.AmazonCommonAuth`? This command belongs to Toolkit, it should not depend on Q states.

Reply:
Yes actually. Any extension can add views, submenus, options, etc to any other extension- which seems silly at first but makes sense in certain scenarios. If we don't add this, then toolkit will put a duplicate Feedback... submenu item in the ellipses menu for Q.